awx/ui_next/src/components/Schedule/ScheduleDetail/ScheduleDetail.jsx

```diff
@@ -140,18 +151,25 @@ function ScheduleDetail({ schedule, i18n }) {
     survey_enabled,
   } = launchData || {};
 
+  const showCredentialsDetail =
+    ask_credential_on_launch && credentials.length > 0;
+  const showInventoryDetail = ask_inventory_on_launch && inventory;
```

Comment:
**showInventoryDetail** equals `null` when **ask_inventory_on_launch** is `true` and **inventory** is `null`. Is that expected?

Reply:
Oof, yes I'll make those same changes for those fields.  Good catch.